app/controllers/finance/lead_providers_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Finance::LeadProvidersController < Finance::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
```

Comment:
We probably should either not skip them, or not include them in the base controller. As I see it, `ensure_finance` is enough for our current needs.